lib/nextstrain/sphinx/theme/static/css/nextstrain.css

```diff
@@ -3,27 +3,41 @@
 @import "theme.css"; /* From sphinx_rtd_theme */
 
 html {
-  --text-color: black;
+  --text-color: #24292e;
   --heading-color: #404040;
   --link-color: #5097ba;
   --sidebar-background-color: #f2f2f2;
+  --content-background-color: #ffffff;
 }
 
 body {
   font-family: Lato, 'Helvetica Neue', sans-serif;
-  font-weight: 300;
+  font-weight: 400;
   color: var(--text-color);
+  line-height: 1.5;
 }
 
 h1, h2, h3, h4, h5, h6, legend, .rst-content .toctree-wrapper p.caption {
   font-family: Lato, 'Helvetica Neue', sans-serif;
-  font-weight: 400;
+  font-weight: 600;
```

Comment:
Happy to see heavier weights! I agree it makes the text more readable. I was matching the initial styles to nextstrain.org/docs.